ext/pdo/pdo_stmt.stub.php

```diff
@@ -29,25 +29,25 @@ public function errorCode() {}
     public function errorInfo() {}
 
     /** @return bool */
-    public function execute(?array $input_parameters = null) {}
+    public function execute(?array $params = null) {}
 
     /** @return mixed */
-    public function fetch(int $mode = PDO::FETCH_BOTH, int $cursor_orientation = PDO::FETCH_ORI_NEXT, int $cursor_offset = 0) {}
+    public function fetch(int $mode = PDO::FETCH_BOTH, int $cursorOrientation = PDO::FETCH_ORI_NEXT, int $cursorOffset = 0) {}
 
     /** @return array */
     public function fetchAll(int $mode = PDO::FETCH_BOTH, mixed ...$args) {}
 
     /** @return mixed */
-    public function fetchColumn(int $index = 0) {}
+    public function fetchColumn(int $column = 0) {}
```

Comment:
I wonder if we should update ext/mysqli $index usage to $column as well (e.g. https://github.com/php/php-src/blob/faea5ab837ab6393c8821f85cf8abe2723593e8e/ext/mysqli/mysqli.stub.php#L340). mysqli uses "field" instead of column in its function names though.

Reply:
I have no idea what the best course of action would be :(